youtube_dl/extractor/picarto.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+import time
+
+from .common import InfoExtractor
+from ..utils import ExtractorError
+from ..compat import compat_str
+
+
+class PicartoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?picarto\.tv/(?P<id>[a-zA-Z0-9]+)'
+    _TEST = {
+        'url': 'https://picarto.tv/setz',
+        'info_dict': {
+            'id': 'Setz',
+            'ext': 'mp4',
+            'title': 're:^Setz [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}$',  # match self._live_title
```

Comment:
To clarify we're trying to match against the output of the `_live_title` function, and not something the site would ever give us. As it's part of the base class and out of my hands, I decided to comment upon it would the `_live_title` ever change.